src/Microsoft.AspNetCore.SystemWebAdapters/HttpException.cs

```diff
@@ -17,50 +18,32 @@ public HttpException(string message) : base(message)
     {
     }
 
-    public HttpException(string message, Exception innerException)
+    public HttpException(String message, Exception innerException)
```

Comment:
Super nit-pick but this change seems unnecessary. We use `string` everywhere else in this file.